src/pages/patientView/mutationalSignatures/SignatureTextBox.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { CSSProperties } from 'react';
+import { action, computed, makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { boolean, number, string } from 'yargs';
+import { IMutationalSignature } from 'shared/model/MutationalSignature';
+import { adjustVisibility } from 'shared/components/alterationsTableUtils';
+import { Modal } from 'react-bootstrap';
+import { ISelectedTrialFeedbackFormData } from 'pages/patientView/trialMatch/TrialMatchTable';
+import Tooltip from 'rc-tooltip';
+
+export interface ISignatureTextBoXProps {
+    height: number;
+    width: number;
+    url: string;
+    description: string;
+    signature: string;
+    show: boolean;
+    onHide: () => void;
+    selectMutationalSignature: (childData: string, visibility: boolean) => void;
+}
+
+export default class SignatureTextBox extends React.Component<
+    ISignatureTextBoXProps,
+    {}
+> {
+    sendData = () => {
+        this.props.selectMutationalSignature(this.props.signature, false);
+    };
+
+    @computed get tooltipInfo() {
+        return (
+            <div
+                style={{
+                    maxWidth: 250,
+                    fontFamily: 'Helvetica Neue, Helvetica, Arial, sans-serif',
```

Comment:
don't change font from standard

Reply:
`SignatureTextBox` is replaced by the standard React ToolTip element.